pkg/innerring/processors/container/process_container.go

```diff
@@ -181,6 +210,14 @@ func (cp *Processor) approveDeleteContainer(e *containerEvent.Delete) {
 		cp.log.Error("could not approve delete container",
 			zap.Error(err),
 		)
+		return
+	}
+
+	cID := cid.ID(e.ContainerID())
+	err = processors.UpdatePlacementVectors(cID, cp.cnrClient, nil, nil)
```

Comment:
Why? It should be deleted by contract right with the container itself.

Reply:
well, this should be done then on the contract side, will do

Reply:
https://github.com/nspcc-dev/neofs-contract/pull/456